_posts/op-ed/2022-11-01-What-is-High-Priority.md

```diff
@@ -0,0 +1,217 @@
+
+{:refdef: style="text-align: center;"}
+![Traditional Risk Management Chart](/images/posts/time-management-risk-traditional.png)
+*Traditional risk management chart*
+{: refdef}
+
+{:refdef: style="text-align: center;"}
+![Agile Risk Management Chart](/images/posts/time-management-risk-agile.png)
+*Agile risk management chart*
+{: refdef}
+
+To combat these risk challenges and ensure we're better serving our stakeholders, most teams I work with have recently implemented a **[Continuous Planning](https://www.umt360.com/blog/continuous-planning)** process. Of course, every implementation of this process is different. But from my perspective, I see it as more agile, an iteration of the agile approach to risk reduction applied to the entire roadmap (rather than a single product).
+
+For each individual case, I'd recommend looking at product release risk history, launch cadence, other [DORA metrics](https://cloud.google.com/blog/products/devops-sre/using-the-four-keys-to-measure-your-devops-performance) and similar data points to decide which process is best. It may be that a long roadmap is better for the team, although that's rarely the case.
+
+The best read on this so far:
+
+  - 🎧📱📚 &nbsp; [The Phoenix Project](https://www.amazon.com/The-Phoenix-Project-audiobook/dp/B00VATFAMI)
+
+{:refdef: style="text-align: center;"}
+![Prioritization](/images/posts/time-management-priority.png)
+{: refdef}
+
+-----
+
+## Tools and hacks
+
+#### 🔕 &nbsp; When you're busy, you're busy
+
+This goes hand in hand with the basic framework I described above. Again, engineering resources don't just simply grow when you need them, so it's not possible to make up engineering time that doesn't exist. That's why I think it's very important that people in the environment understand the constraints of each team member… to be clear, I have limited time to work, limited team resources, and a pretty fixed schedule. This framework ensures that I finish my work on time (or almost on time), which in turn guarantees that I have time for a life outside of work as well.
+
+I'm a firm believer that all requests, inquiries, and questions (personal or professional) deserve a response -- at least to communicate that I'm busy. It takes almost no time to reply _"I'm busy right now, let me get back to you later"_, so I try to at least do that whenever possible. But if I'm having a really busy day, I'll set my [Slack status to busy](https://slack.com/help/articles/214908388-Pause-notifications-with-Do-Not-Disturb) -- then the messages often don't come at all, because people immediately see that I'm busy.
+
+{:refdef: style="text-align: center;"}
+![Slack Profile](/images/posts/time-management-slack-profile.png)
+*Example of a Slack profile with DND mode and Calendar status*
+{: refdef}
+
+#### 📅 &nbsp; About meetings…
```

Comment:
The content of this section is good, but I'm not sure if this is the right place for it. Or if you want to keep it here, add a sentence or two about how it relates to prioritization

Reply:
Actually, I think it might fit better into the Inbox zero post because both provide practical tips, whereas this post is a bit higher level